app/models/person.rb

```diff
@@ -218,7 +218,7 @@ def self.username_blacklist
   end
 
   def self.username_available?(username, community_id)
-    !username.in?(USERNAME_BLACKLIST) &&
+    !username.downcase.in?(USERNAME_BLACKLIST) &&
```

Comment:
Maybe turn this the other way around:

```ruby
USERNAME_BLACKLIST.include?(username.downcase)
```

The `in?` method actually does excatly that an is a Rails addition to Object. Personally, I'd prefer the use of core Ruby methods.

Maybe even consider adding to the end of line 158 `to_set`. There's no need for duplicate entries in the blacklist and `include?` is faster on sets (though here I doubt it matters much).

```ruby
 USERNAME_BLACKLIST = YAML.load_file("#{Rails.root}/config/username_blacklist.yml").to_set
```